src/ui/comment/CommentAddOrEditForm.tsx

```diff
@@ -126,12 +133,19 @@ function CommentAddOrEditFormContent({
 	);
 }
 
+type Props = {
+	reviewAnnotation: CardContentComponentProps['reviewAnnotation'];
```

Comment:
Change it to `AnnotationDataModel` instead of `ReviewAnnotation` type.

Reply:
That would cause other errors. I think the best option will be to add targets attribute as optional to the `ReviewAnnotation` type.